Localizations/en_US.json

```diff
@@ -16,18 +16,18 @@
     "asset_tab_image": "Image",
     "asset_tab_information": "Information",
     "asset_tab_text": "Text",
-    "asset_tab_yaml": "Yaml",
+    "asset_tab_yaml": "YAML",
```

Comment:
Acronyms in English are generally correct, regardless of letter casing.

Reply:
Yes, but I thought that this way it could be more consistent; usually, names and file types are all in uppercase. However, if you prefer, I can revert it to how it was before.

Reply:
In general, I prefer Pascal when the acronym is used as a noun and lowercase when used as an adjective. UPPERCASE feels too shouty.

Reply:
Sure, I'll leave it as it was before.